test/dialect/AIE/nd-dma-wrong-rank.mlir

```diff
@@ -9,6 +9,7 @@
 //===----------------------------------------------------------------------===//
 
 // RUN: aie-opt --verify-diagnostics %s
+// XFAIL: *
```

Comment:
Should this test and the next one fail or catch an intended error?

Reply:
They pass because the error disallowing ND DMA transformations for any memref type other than 1xi32 is disabled